webapp/channels/src/components/emoji_picker/emoji_picker.tsx

```diff
@@ -159,6 +160,23 @@ const EmojiPicker = ({
         return emoji;
     };
 
+    const selectFirstEmoji = (emojiPositions: EmojiPosition[]) => {
+        const firstEmoji = emojiPositions[0];
+        if (!firstEmoji) {
+            return;
+        }
+
+        const {rowIndex, emojiId} = firstEmoji;
+        const cursorEmoji = getEmojiById(firstEmoji.emojiId);
```

Comment:
We're already destructuring props above, so calling this one as `firstEmoji.emojiId` seems redundant.

I'm okay with either or, but if we're destructuring already we should use those.
```suggestion
        const cursorEmoji = getEmojiById(emojiId);
```